api/src/main/java/io/minio/PostPolicy.java

```diff
@@ -167,14 +167,6 @@ public String bucketName() {
   }
 
   /**
-   * Returns form data of this post policy.
-   */
-  public Map<String,String> formData(String accessKey, String secretKey)
```

Comment:
> However, this function doesn't seem to be working properly before, as it uses "us-east-1" by default unless another API was called before and updated the region cache with the correct value.

I couldn't able to understand this.  When `MinioClient.presignedPostPolicy(PostPolicy)` method is called, it does `updateRegionCache(policy.bucketName());` then calls `policy.formData(this.accessKey, this.secretKey);` where bucket region cache is used.

Do you say this is not working?